lib/slack/web/docs/users.lookupByEmail.json

```diff
@@ -0,0 +1,15 @@
+{
+	"desc": "Retrieve a single user by looking them up by their registered email address. Requires `users:read.email`.",
+
+	"args": {
+		"email": {
+			"type"		: "string",
+			"required"	: true,
+			"desc"		: "User's email address"
+		}
+	},
+
+	"errors": {
+		"user_not_found"	: "Value passed for `user` was invalid."
```

Comment:
It's probably a good idea to include `missing_scope` since `users:read.email` being a required scope permission is specifically called out in the docs for this method.

Reply:
Ah good call! I'll add it.